x-pack/plugins/ml/public/application/jobs/new_job/job_from_lens/utils.ts

```diff
@@ -44,8 +46,32 @@ export const COMPATIBLE_SPLIT_FIELD_TYPES: DataType[] = [
   KBN_FIELD_TYPES.IP,
 ];
 
+export async function redirectToADJobWizards(
+  embeddable: Embeddable,
+  layerIndex: number,
+  share: SharePluginStart
+) {
+  const { query, filters, to, from, vis } = getJobsItemsFromEmbeddable(embeddable);
+  const locator = share.url.locators.get(ML_APP_LOCATOR);
+
+  const url = await locator?.getUrl({
+    page: ML_PAGES.ANOMALY_DETECTION_CREATE_JOB_FROM_LENS,
+    pageState: {
+      vis: vis as any,
+      from,
+      to,
+      query,
+      filters,
+      layerIndex,
+    },
+  });
+
+  window.open(url, '_blank');
```

Comment:
Please use `navigateToUrl` from `CoreStart['application']`

Reply:
This link needs to open in a new tab, from what I can tell `navigateToUrl` does not allow this as an option.